docs/dev/python/user-guide/table/10_minutes_to_table_api.zh.md

```diff
@@ -0,0 +1,719 @@
+
+In fact, this shows the change logs received by the print sink.
+The output format of a change log is:
+{% highlight python %}
+{subtask id}> {message type}{string format of the value}
+{% endhighlight %}
+For example, "2> +I(4,11)" means this message comes from the 2nd subtask, and "+I" means it is an insert message. "(4, 11)" is the content of the message.
+In addition, "-U" means a retract record (i.e. update-before), which means this message should be deleted or retracted from the sink. 
+"+U" means this is an update record (i.e. update-after), which means this message should be updated or inserted by the sink.
+
+So, we get this result from the change logs above:
+
+{% highlight text %}
+(4, 11)
+(2, 15) 
+(3, 19)
+{% endhighlight %}
+
+### Mixing the Table API and SQL
+
+The `Table` objects used in Table API and the tables used in SQL can be freely converted to each other.
+
+The following example shows how to use a `Table` object in SQL:
+
+{% highlight python %}
+
+# create a sink table to emit results
+table_env.execute_sql("""
+    CREATE TABLE table_sink (
+        id BIGINT, 
+        data VARCHAR 
+    ) WITH (
+        'connector' = 'print'
+    )
+""")
+
+# convert the Table API table to a SQL view
+table = table_env.from_elements([(1, 'Hi'), (2, 'Hello')], ['id', 'data'])
+table_env.create_temporary_view('table_api_table', table)
+
+# emit the Table API table
+table_env.execute_sql("INSERT INTO table_sink SELECT * FROM table_api_table") \
+    .get_job_client().get_job_execution_result().result()
+
+{% endhighlight %}
+
+The result is:
+
+{% highlight text %}
+6> +I(1,Hi)
+6> +I(2,Hello)
+{% endhighlight %}
+
+And the following example shows how to use SQL tables in the Table API:
+
+{% highlight python %}
+
+# create a sql source table
+table_env.execute_sql("""
+    CREATE TABLE sql_source (
+        id BIGINT, 
+        data TINYINT 
+    ) WITH (
+        'connector' = 'datagen',
+        'fields.id.kind'='sequence',
+        'fields.id.start'='1',
+        'fields.id.end'='4',
+        'fields.data.kind'='sequence',
+        'fields.data.start'='4',
+        'fields.data.end'='7'
+    )
+""")
+
+# convert the sql table to Table API table
+table = table_env.from_path("sql_source")
+
+# or create the table from a sql query
+table = table_env.sql_query("SELECT * FROM sql_source")
+
+# emit the table
+table.to_pandas()
+
+{% endhighlight %}
+
+{% top %}
+
+Emit Results
+------------
+
+### Capturing Data in a Variable
+
+You can call the "to_pandas" method to emit the data from a `Table` object to a pandas DataFrame:
+
+{% highlight python %}
+
+table = table_env.from_elements([(1, 'Hi'), (2, 'Hello')], ['id', 'data'])
+table.to_pandas()
+
+{% endhighlight %}
+
+The result is:
+
+{% highlight text %}
+   id   data
+0   1     Hi
+1   2  Hello
+{% endhighlight %}
+
+Note that "to_pandas" is not supported by the flink planner, and not all data types can be emitted to pandas DataFrames.
+
+### Emitting Results to One Sink Table
```

Comment:
```suggestion
### Emit Results to One Sink Table
```
What about remove the `ing` suffix. Personally I think it's not needed. Same for the other places.

Reply:
It seems that we have different opinions here. What do you think? @alpinegizmo I think "Emitting" or "Emit" are both grammatically reasonable, but we should have a consistent style throughout the document.

Reply:
Yes, both are grammatically correct. I prefer "Emitting" because it is softer in tone. I am being shown how to do something, rather than being ordered to do it. "Emit" is too direct, in my opinion.

Reply:
@alpinegizmo Thanks a lot for the explanation. I'm OK with "Emit". Just propose this suggestion as I noticed that there is no **ing** suffix in the following doc: https://ci.apache.org/projects/flink/flink-docs-master/dev/table/common.html.

However, both cases exist throughout the Flink documentation. I'm OK with both of them as long as we make sure that the styles keep consistent.

Reply:
In that case, I prefer "Emit" for the sake of consistency.

Reply:
Thanks @alpinegizmo !